pom.xml

```diff
@@ -74,7 +74,7 @@
     <dropwizard.version>1.3.12</dropwizard.version>
     <horizon.version>0.1.2</horizon.version>
     <mesos.docker.tag>1.7.1</mesos.docker.tag> <!-- 1.8.0 not available yet, still compatible -->
```

Comment:
This is the tag we'd have to update for the SingularityExecutor/Dockerfile one. If I remember correctly, this controls the output tag from the maven build